client/response.go

```diff
@@ -77,11 +77,26 @@ func (c *Client) Guilds() ([]Channel, error) {
 
 func (c *Client) GuildMessages(channel *Channel, me *Me, seek *int) (*Messages, error) {
 	endpoint := fmt.Sprintf(endpoints["guild_msgs"], channel.ID, me.ID, *seek, messageLimit)
+	endpoint_ := fmt.Sprintf(endpoints["guild_channels"], channel.ID)
```

Comment:
Style: underscores in variable names in Go are not too common. Try to think of an appropriate name for these variables.